akd/src/append_only_zks.rs

```diff
@@ -297,26 +266,27 @@ impl Azks {
         &self,
         storage: &S,
         label: NodeLabel,
-        epoch: u64,
+        _epoch: u64,
```

Comment:
nit: If we no longer need the epoch for these proofs (and it isn't going to be put back later) can we clean up the function signature?